platform/linux-generic/include/odp_packet_internal.h

```diff
@@ -57,6 +57,12 @@ typedef struct {
 
 	/* offset to L4 hdr (TCP, UDP, SCTP, also ICMP) */
 	uint16_t l4_offset;
+
+	/* Partial sum for L4 checksumming */
+	uint32_t l4_part_sum;
+
+	/* L4 checksum */
+	uint32_t l4_sum;
```

Comment:
This is performance sensitive part of packet header. Data before and after this point is needed frequently. And small modifications here have made +-10% performance impact previously.

When L4 checksum is calculated in SW, we are already ruined performance, so these two could be located in the bottom of the struct, e.g. just before crypto results.